docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -190,6 +190,26 @@ include::partial$selenium-properties.adoc[]
 |proxy
 |empty
 |Provides possibility to attach screenshot for failed proxy steps
+
+|proxy.enabled
+|true/false
+|false
+|Enabled proxy for the run
+
+|proxy.host
+|any host name resolving on the machine
+|empty
+|Overrides the host which will be used by proxy
+
+|proxy.ports
+|10000-10005
+|empty
+|Defines ports range which could be used for proxy
+
+|selenium.proxy.host
```

Comment:
`selenium.proxy.host` -> `proxy.connectable-host` ? that's a naming from BrowserUp proxy

> connectableAddress the network address the Selenium Proxy will use to reach this BrowserUpProxy instance